lib/common/api/module-list.js

```diff
@@ -0,0 +1,11 @@
+// Common modules, please sort with alphabet order.
```

Comment:
Since we are moving this to a new location mind be worth updating this to be `alphabetically` instead of `alphabet`.